### ISSUE TITLE
the bot doesn't find the product (US)

### ISSUE THREAD

**Issue by mef98** [val92130/Supreme-Auto-Checkout#169]
the bot doesn't find the product and continues to refresh the supreme page.
it works perfectly in UE, not in US!
what do i have to change in the bundle.js to make the bot working?

**Comment by hasanahmed1** [comment]
Yes, i am having same issue. In AutoCop, the bot just infinitely refreshes on Supreme US site.

**Comment by ghost** [comment]
same problem. please fix it ^_^
